spec.go

```diff
@@ -39,7 +39,7 @@ func specification() sdk.Specification {
 		Summary:     "An Google Cloud Storage Source and Destination Connector for Conduit, Written in Go.",
 		Description: "Real time data transmission with google cloud storage",
 		Version:     version,
-		Author:      "Santosh Kumar Gajawada",
+		Author:      "InfraCloud Technologies & Meroxa, Inc.",
```

Comment:
oh you got the update too, you can overwrite the one I did, "&" looks nicer

Reply:
OK, I'll do that. Maybe you merge your PR first, and then I merge this one, with the overwrite?